numpy/lib/arraypad.py

```diff
@@ -138,8 +138,8 @@ def _append_const(arr, pad_amt, val, axis=-1):
         return np.concatenate((arr, np.zeros(padshape, dtype=arr.dtype)),
```

Comment:
While you're here, you could remove this branch too

Reply:
Sure.